test/e2e/runner/setup.go

```diff
@@ -86,7 +86,8 @@ func Setup(testnet *e2e.Testnet) error {
 		if err != nil {
 			return err
 		}
-		cfg.IPFS.ConfigRootPath = filepath.Join(nodeDir, "ipfs")
+		// todo(evan): the path should be a constant
+		cfg.IPFS.ConfigRootPath = filepath.Join(nodeDir, ".ipfs")
```

Comment:
This doesn't matter much. The nodeDir in the e2e is ephemeral anyways.

Reply:
yeah, in this case I was just being consistent with the directory that the plugin uses. https://github.com/lazyledger/lazyledger-core/blob/c07235d14117e60b72626a9138c7fbcdc9468b7b/p2p/ipld/plugin/Makefile#L8